src/ui/units/wizard/components/ValuesList/ValuesList.tsx

```diff
@@ -211,27 +211,30 @@ class ValuesList extends React.Component<Props, State> {
 
         try {
             let distincts;
-            let values;
+            let values: string[] = [];
+
             if (externalDistincts) {
                 values = externalDistincts[item.guid];
             } else {
                 distincts = await this.getDistincts();
                 values = this.getValuesFromDistincts(distincts);
             }
+
             const useSuggest = values.length === VALUES_LOAD_LIMIT;
 
+            const oldItemGuid = this.state.itemGuid;
+            const newItemGuid = item.guid;
+
             this.setState({
                 values,
                 useSuggest,
-                itemGuid: item.guid,
                 error: null,
-            });
-            this.setState({
                 fetching: false,
+                itemGuid: newItemGuid,
             });
 
             const shouldClearPalette =
-                this.state.itemGuid !== null && this.state.itemGuid !== item.guid;
+                Boolean(oldItemGuid) && Boolean(newItemGuid) && newItemGuid !== oldItemGuid;
```

Comment:
guid can't be equal 0 or an empty string?

Reply:
No, guid is always non-empty string